web/priv/js_effects/word_game.js

```diff
@@ -0,0 +1,3151 @@
+            "taper",
+            "taper",
+            "husky",
+            "brave",
+            "filet",
+            "slime",
+            "glint",
+            "tread",
+            "tread",
+            "regal",
+            "guest",
+            "every",
+            "murky",
+            "share",
+            "spore",
+            "hoist",
+            "buxom",
+            "inner",
+            "otter",
+            "dimly",
+            "level",
+            "level",
+            "donut",
+            "stilt",
+            "arena",
+            "sheet",
+            "sheet",
+            "fancy",
+            "fancy",
+            "pearl",
+            "silly",
+            "porch",
+            "porch",
+            "sepia",
+            "amble",
+            "amble",
+            "bread",
+            "friar",
+            "friar",
+            "dairy",
+            "quill",
+            "cross",
+            "cross",
+            "cross",
+            "shear",
+            "posit",
+            "blank",
+            "villa",
+            "villa",
+            "villa",
+            "villa",
+            "which",
+            "among",
+            "among",
+            "shell",
+            "would",
+            "algae",
+            "algae",
+            "algae",
+            "algae",
+            "amuse",
+            "amuse",
+            "copse",
+            "copse",
+            "copse",
+            "pouch",
+            "ascot",
+            "plane",
+            "crown",
+            "crown",
+            "crown",
+            "relay",
+            "abide",
+            "viola",
+            "viola",
+            "straw",
+            "straw",
+            "straw",
+            "amass",
+            "third",
+            "trick",
+            "tutor",
+            "tutor",
+            "blurb",
+            "blurb",
+            "disco",
+            "where",
+            "where",
+            "beach",
+            "beach",
+            "comic",
+            "clued",
+            "clued",
+            "clued",
+            "graze",
+            "graze",
+            "graze",
+            "graze",
+            "graze",
+            "graze",
+            "lurid",
+            "steel",
+            "halve",
+            "halve",
+            "vinyl",
+            "utile",
+            "utile",
+            "adage",
+            "worry",
+            "worry",
+            "local",
+            "trade",
+            "trade",
+            "ashen",
+            "modal",
+            "gaunt",
+            "clove",
+            "clove",
+            "adorn",
+            "roast",
+            "speck",
+            "speck",
+            "speck",
+            "grunt",
+            "snoop",
+            "snoop",
+            "party",
+            "party",
+            "emcee",
+            "array",
+            "south",
+            "vapid",
+            "jelly",
+            "skulk",
+            "angst",
+            "tubal",
```

Comment:
"Tubal" also  bit on the edge, typically cited with fallopian tubes etc. Some may be sensitive. let's drop it.